feat: ✨ post about poster and submission to OIS (#215)
seedcase-project/community: posts/conference-ois-2026/index.qmd

---
title: "Open Innovation in Science: Research Conference 2026"
description: "May 11-13, 2026 in Copenhagen, Denmark."
date: "2026-05-11"
---

We don't often find conferences that are a good fit for our work, but we
stumbled on the [Open Innovation in Science: Research Conference
2026](https://ois-research-conference.org), which aligned very well to
what we do. The conference focuses on bringing together people who want
to adopt or already use open and collaborative approaches in science,

> especially focosing on innovation activities between industry and

Reviewer: ```suggestion
with a focus on innovation activities between industry and
```

academia. So we sent a [submission](submission.qmd) and were accepted to
present a poster at the conference.

> This poster is what we presented at the conference.

Reviewer: ```suggestion
We presented the poster below at the conference.
```
